test/e2e/scripts/utils/local_system.py

```diff
@@ -33,22 +32,20 @@ def find_free_port(start: int, step: int):
 
 
 @allure.step('Kill process')
-def kill_process(pid, sig: signal.Signals = signal.SIGKILL):
-    LOG.debug('Sending %s to %d process', sig.name, pid)
-    try:
-        p = psutil.Process(pid)
+def kill_process(pid):
+    LOG.debug(f'Terminating process {pid}')
 
-    except ProcessLookupError as err:
-        LOG.error('Failed to find process %d: %s', pid, err)
-        raise err
-    for i in range(2):
-        try:
-            LOG.warning('Waiting for process to exit: %d', pid)
-            p.kill()
-            if not p.is_running():
-                break
-        except RuntimeError:
-            raise ('Failed to kill process: %d' % pid)
+    current_platform = platform.system()
```

Comment:
We have already platforms in configs.system.py file - https://github.com/status-im/status-desktop/blob/master/test/e2e/configs/system.py and use it in many tests.
So maybe it would be better to use it here as well?

Reply:
ya, good point, i committed some changes , lemme know wdyt @Valentina1133